messages/jsonschema/GherkinDocument.json

```diff
@@ -338,6 +338,7 @@
       "required": [
         "location",
         "keyword",
+        "keywordType",
```

Comment:
In terms of interoperability, I'm wary of making this required and breaking backwards compatibility with older streams (e.g. an older version of cucumber reporting test results to a newer version of a formatter).

Is it strictly necessary, or is there a path where we can introduce it as optional?

Reply:
I appreciate this shifts some complexity into the consuming cucumber implementations, but they're already handling streams without these extra fields in, so can presumably fall back to existing behaviour

Reply:
This is actually discussed :D

Reply:
Ah can you point me to the right place?

Reply:
Ah indeed @aslakhellesoy made the point